pkg/controller/kogitoinfra/infinispan/infinispan_credentials.go

```diff
@@ -45,55 +36,6 @@ type Credential struct {
 	Password string `yaml:"password"`
 }
 
-// generateDefaultCredentials will generate random Kogito credentials to inject into Infinispan Operator
-// Deprecated: not in use right now because of this issue: https://github.com/infinispan/infinispan-operator/issues/211
-func generateDefaultCredentials() (yamlFile, fileMD5 string, err error) {
-	credentials := make([]Credential, len(defaultInfinispanUsers))
-	for i, user := range defaultInfinispanUsers {
-		credentials[i].Password = generateRandomPassword()
-		credentials[i].Username = user
-	}
-	identity := Identity{Credentials: credentials}
-
-	data, err := yaml.Marshal(&identity)
-	if err != nil {
-		return "", "", err
-	}
-	return string(data), getMD5FromBytes(data), nil
-}
-
-// generateRandomPassword generates a random password to inject into infinispan identities
-// Deprecated: not in use right now because of this issue: https://github.com/infinispan/infinispan-operator/issues/211
-func generateRandomPassword() string {
-	b := make([]byte, passwordSize)
-	for i := range b {
-		b[i] = passwordAllowedBytes[rand.Int63()%int64(len(passwordAllowedBytes))]
-	}
-	return string(b)
-}
-
-func getMD5FromBytes(data []byte) string {
-	hash := md5.New()
-	hash.Write(data)
-	return hex.EncodeToString(hash.Sum(nil))
-}
-
-// hasKogitoUser will find if the kogitoUser is set in the identities file. If we can't decode the file for some reason, will return false
-// something odd could be happened to the file, so let's others decide to regenerate the file. An error message will be displayed in logs
-func hasKogitoUser(secretFileData []byte) bool {
```

Comment:
same here for the above functions (not used)?

Reply:
Yes, not used as well